vcf2db.py

```diff
@@ -41,9 +43,39 @@
         'gt_types': sql.SmallInteger,
         }
 
+"""
+Under Python 2 this function b() will return the string you pass in, ready for use as binary data:
+>>> b('GIF89a')
+'GIF89a'
+
+While under Python 3 it will take a string and encode it to return a bytes object:
+>>> b('GIF89a')
+b'GIF89a'
+"""
+# Python2
+if sys.version_info < (3,):
+    from itertools import imap as map
+    ESCAPE = "string_escape"
+    def b(x):
+        return x
+# Python3
+else:
+    ESCAPE = "unicode_escape"
+    unicode = str
+    buffer = memoryview
+    def b(x):
+        return x.encode('ISO-8859-1')
```

Comment:
why this encoding and why `b()` as the function name? is there something more descriptive?